VideoLocker/src/test/java/org/edx/mobile/test/BaseTestCase.java

```diff
@@ -25,24 +24,27 @@
 import java.util.HashMap;
 import java.util.Map;
 import java.util.Set;
+import java.util.TimeZone;
 
 import roboguice.RoboGuice;
 
 @Ignore
 public abstract class BaseTestCase extends BaseTest {
-
     protected final Logger logger = new Logger(getClass().getName());
 
     protected Context context;
     protected Config config;
     protected CustomGuiceModule module;
+
     @Before
     public void setUp() throws Exception {
         context = RuntimeEnvironment.application;
         config = new Config(generateConfigProperties());
 
         module = new CustomGuiceModule();
         glueInjections();
+        // Set time zone to UTC to make time-based tests predictable
```

Comment:
'UTC' should be changed to 'a constant value', since we're not referring to UTC directly here.